rpc/src/v1/impls/secretstore.rs

```diff
@@ -21,7 +21,7 @@ use std::sync::Arc;
 
 use crypto::DEFAULT_MAC;
 use ethkey::Secret;
-use ethcore::account_provider::AccountProvider;
+use accounts::AccountProvider;
```

Comment:
Just curios - are we going to leave account management under `accounts` feature forever (i.e. for some long period), or it'll be totally purged from the codebase in the nearest future? In the latter case, we'll need to replace all `address: H160, password: Password` in SS RPCs with something like `secret: H256` or do something else.

Reply:
It's not defined yet, perhaps we are going to move some functionality to the proxy, but if we decide to remove accounts completely we need to provide some non-account-based alternatives like using secret directly. Don't know the definitive answer yet though.